HCaptcha/Assets/hcaptcha.html

```diff
@@ -30,6 +30,15 @@
         window.webkit.messageHandlers.hcaptcha.postMessage(value);
       };
 
+      const getExecuteOpts = function() {
+        let opts;
+        const rqdata = "${rqdata}";
+        if (rqdata) {
+          opts = {rqdata};
```

Comment:
```suggestion
          opts = {'rqdata': rqdata};
```

The expected usage is: `hcaptcha.execute({ rqdata: "<rqdata_string>" });`

Reply:
Good catch! This is exactly why I asked about ES6. Because this approach completely valid in ES6

I agree with your suggestion